package-lock.json

```diff
@@ -4,6 +4,28 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "@ampproject/remapping": {
```

Comment:
It hurts my heart that this is such a massive change.

Reply:
Fine for there to be a big change when adding a package like Jest but you've included TryNow-internal CodeArtifact URLs in this diff here that won't be usable by the general public.

You need to revert and re-generate `package-lock.json` but without the CodeArtifact configuration set (`npm config delete registry` I think)